tests/avifyuv.c

```diff
@@ -106,140 +113,175 @@ int main(int argc, char * argv[])
                     continue;
                 }
 
-                const avifMatrixCoefficients matrixCoeffsList[] = { AVIF_MATRIX_COEFFICIENTS_BT709,
-                                                                    AVIF_MATRIX_COEFFICIENTS_BT601,
-                                                                    AVIF_MATRIX_COEFFICIENTS_BT2020_NCL,
-                                                                    AVIF_MATRIX_COEFFICIENTS_IDENTITY,
-                                                                    AVIF_MATRIX_COEFFICIENTS_YCGCO };
-                const int matrixCoeffsCount = (int)(sizeof(matrixCoeffsList) / sizeof(matrixCoeffsList[0]));
-
-                for (int matrixCoeffsIndex = 0; matrixCoeffsIndex < matrixCoeffsCount; ++matrixCoeffsIndex) {
-                    avifMatrixCoefficients matrixCoeffs = matrixCoeffsList[matrixCoeffsIndex];
-
-                    int dim = 1 << rgbDepth;
-                    int maxDrift = 0;
-
-                    avifImage * image = avifImageCreate(dim, dim, yuvDepth, AVIF_PIXEL_FORMAT_YUV444);
-                    image->colorPrimaries = AVIF_COLOR_PRIMARIES_BT709;
-                    image->transferCharacteristics = AVIF_TRANSFER_CHARACTERISTICS_SRGB;
-                    image->matrixCoefficients = matrixCoeffs;
-                    image->yuvRange = AVIF_RANGE_FULL;
-                    avifImageAllocatePlanes(image, AVIF_PLANES_YUV);
-
-                    avifRGBImage srcRGB;
-                    avifRGBImageSetDefaults(&srcRGB, image);
-                    srcRGB.format = AVIF_RGB_FORMAT_RGB;
-                    srcRGB.depth = rgbDepth;
-                    avifRGBImageAllocatePixels(&srcRGB);
-
-                    avifRGBImage dstRGB;
-                    avifRGBImageSetDefaults(&dstRGB, image);
-                    dstRGB.format = AVIF_RGB_FORMAT_RGB;
-                    dstRGB.depth = rgbDepth;
-                    avifRGBImageAllocatePixels(&dstRGB);
-
-                    uint64_t driftPixelCounts[MAX_DRIFT];
-                    for (int i = 0; i < MAX_DRIFT; ++i) {
-                        driftPixelCounts[i] = 0;
-                    }
-
-                    for (int r = 0; r < dim; ++r) {
-                        if (verbose) {
-                            printf("[%4d/%4d] RGB depth: %d, YUV depth: %d, matrixCoeffs: %d\r", r + 1, dim, rgbDepth, yuvDepth, matrixCoeffs);
+                const avifCICP CICPsList[] = {
```

Comment:
Nit: Variable names in libavif generally start with a lowercase letter, so this should be named "cicpList". Since "List" implies plural, it is not necessary to add "s" after "cicp".

Optional: Since this variable is a constant, it is common to name constants as kFooBar. Under that naming convention, this constant would be named kCicpList. I don't know if libavif uses this naming convention for constants.

Reply:
Done. Looks like libavif doesn't use the k-convention.

Reply:
Correct, libavif doesn't use Hungarian notation.